web/src/features/exchanges/ExchangeArrow.tsx

```diff
@@ -26,124 +23,82 @@ function ExchangeArrow({
     r: rotation + (netFlow > 0 ? 180 : 0),
   };
 
-  // Don't render if the flow is very low ...
-  if (absFlow < 1) {
-    return null;
-  }
-
-  // ... or if the arrow would be very tiny ...
-  if (transform.k < 0.1) {
-    return null;
-  }
-
-  // ... or if it would be rendered outside of viewport.
-  if (transform.x + 100 * transform.k < 0) {
+  if (
+    // or if the arrow would be very tiny
+    transform.k < 0.1 ||
+    // or if it would be rendered outside of viewport.
+    transform.x + 100 * transform.k < 0 ||
+    transform.x - 100 * transform.k > viewportWidth ||
+    transform.y - 100 * transform.k > viewportHeight
+  ) {
     return null;
   }
 
-  if (transform.y + 100 * transform.k < 0) {
-    return null;
-  }
+  const setIsMoving = useSetAtom(mapMovingAtom);
 
-  if (transform.x - 100 * transform.k > viewportWidth) {
-    return null;
-  }
+  useEffect(() => {
+    const cancelWheel = (event: Event) => event.preventDefault();
+    const exchangeLayer = document.querySelector('#exchange-layer');
+    if (!exchangeLayer) {
+      return;
+    }
+    exchangeLayer.addEventListener('wheel', cancelWheel, {
+      passive: true,
+    });
+    return () => exchangeLayer.removeEventListener('wheel', cancelWheel);
+  }, []);
 
-  if (transform.y - 100 * transform.k > viewportHeight) {
-    return null;
-  }
+  const prefix = colorBlindMode ? 'colorblind-' : '';
+  const intensity = quantizedCo2IntensityScale(co2intensity);
+  const speed = quantizedExchangeSpeedScale(absFlow);
+  const imageSource = resolvePath(
+    `images/arrows/${prefix}arrow-${intensity}-animated-${speed}`
+  ).pathname;
 
   return (
-    <>
-      <MobileTooltipWrapper
-        tooltipClassName="flex max-h-[256px] max-w-[512px]"
-        tooltipContent={<MobileExchangeTooltip exchangeData={data} />}
-        side="top"
-        sideOffset={10}
-      >
-        <picture
-          id={key}
-          className="md:hidden"
-          style={{
-            transform: `translateX(${transform.x}px) translateY(${transform.y}px) rotate(${transform.r}deg) scale(${transform.k})`,
-            cursor: 'pointer',
-            overflow: 'hidden',
-            position: 'absolute',
-            pointerEvents: 'all',
-            imageRendering: 'crisp-edges',
-            left: '-25px',
-            top: '-41px',
-          }}
-          onWheel={() => setIsMoving(true)}
-        >
-          <source srcSet={`${imageSource}.webp`} type="image/webp" />
-          <img src={`${imageSource}.gif`} alt="" draggable={false} />
-        </picture>
-      </MobileTooltipWrapper>
-      <TooltipWrapper
-        tooltipClassName="max-h-[256px] max-w-[512px] top-[-76px] hidden md:flex"
-        tooltipContent={<ExchangeTooltip exchangeData={data} />}
-        side="right"
-        sideOffset={10}
+    <TooltipWrapper
+      tooltipClassName="max-h-[256px] max-w-[512px] top-[-76px]"
+      tooltipContent={<ExchangeTooltip exchangeData={data} />}
+      side="right"
```

Comment:
Should the side be `top` for mobile and `side` on desktop like it was before?

Reply:
I'll double check!